### PR TITLE
[SW-2272] Add test for setH2ONodeLogLevel

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: core/src/test/scala/ai/h2o/sparkling/H2OContextTestSuite.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@RunWith(classOf[JUnitRunner])
+class H2OContextTestSuite extends FunSuite with SharedH2OTestContext {
+
+  override def createSparkSession(): SparkSession = sparkSession("local[*]")
+
+  test("setH2OLogLevel") {
+    hc.setH2OLogLevel("DEBUG")
+    assert(hc.getH2OLogLevel() == "DEBUG")
+  }
+
```
Comment (mn-mikke) [inline core/src/test/scala/ai/h2o/sparkling/H2OContextTestSuite.scala:36]: nit: extra line

**Comment by mn-mikke** [comment]
Was there any troubles with setting log level?

**Comment by jakubhava** [comment]
Yup, not on master but https://github.com/h2oai/sparkling-water/pull/2138. User on gitter was calling `setH2OLogLevel` in external backend but the LogUtil class was missing in the extensions. This is no longer case on master as we user rest api.

**Comment by jakubhava** [comment]
But when fixing in on rel branch, I created a test on master as well to have this feature verified

File: core/src/test/scala/ai/h2o/sparkling/H2OContextTestSuite.scala (hunk 1 above)
Comment (jakubhava) [reply core/src/test/scala/ai/h2o/sparkling/H2OContextTestSuite.scala]: thank you!

**Pull request merged**